src/Bynder/Api/Impl/OAuth2/RequestHandler.php

```diff
@@ -37,7 +37,18 @@ public function getAccessToken($code)
 
     protected function sendAuthenticatedRequest($requestMethod, $uri, $options = [])
     {
+        $updatedHeaders = ['headers'=> [
+            'User-Agent' => 'bynder-php-sdk/' . $this->configuration->getSdkVersion()
+            ]
+        ];
+        
         $this->configuration->refreshToken($this->oauthProvider);
+        if(!is_null($options['headers'])) {
+            $updatedHeaders = ['headers' => array_merge($options['headers'],
+                                                        $updatedHeaders['headers']
+                                                        )
+                                ];
+        }
```

Comment:
Can't we do something like this? (not sure about the right syntax for `$options['headers'] || []` here, but at least let it default to `[]` somehow.

```suggestion
        $updatedHeaders = ['headers' => array_merge(
            $options['headers'] || [],
            ['User-Agent' => 'bynder-php-sdk/' . $this->configuration->getSdkVersion()]
        )];
```